### PR TITLE
about actress added

### PR DESCRIPTION
**Pull request by rohitkabra13**

### CODE REVIEW

**Comment by rohitkabra13** [comment]
actress info added

File: git-mini.html [hunk 1]
```diff
@@ -13,7 +13,9 @@
                     <li><a href="#actor's bio">Home</a></li>
                     <li><a href="#about">About</a></li>
                     <li><a href="#contact info">Contact</a></li>
+                    <li><a href="#about actress">About</a></li>
```
Comment (dhrupad09) [inline git-mini.html]: I like your changes

**Review by dhrupad09 — COMMENTED** [review]
I like your commit

File: git-mini.html (hunk 1 above)
Comment (dhrupad09) [reply git-mini.html]: okay done

**Review by dhrupad09 — COMMENTED** [review]
okay done

**Pull request merged**